packages/frontend/src/translations/en.global.json

```diff
@@ -530,6 +530,21 @@
         "or": "or",
         "title": "You’ve received a NEAR drop!"
     },
+    "NFTDetail": {
+        "owner": "OWNER",
+        "transfer": "Transfer"
+    },
+    "NFTTransfer": {
+      "transferNft": "Transfer NFT",
+      "enterReceipt": "Enter a recipient address, then proceed to confirm your transaction.",
+      "cancel": "Cancel",
+      "next": "Next",
+      "confirm": "Confirm",
+      "transactionComplete": "Transaction Complete!",
+      "viewReceipt": "View Receipt",
+      "continue": "Continue",
+      "youSent": "You sent <span class='color-blue'>${title}</span> to <span class='color-green'>${receiverId}</span>"
```

Comment:
There's a bug with the translation package we're using where automatic fallback to english isn't working whenever the translation string contains `data`. The current solution is to just copy the added translation strings in this `en.global.json` file and paste it in the other `json` files under `/translations`. Can you please do that in this PR? 🙏

Reply:
done, please check if I did this correctly

Reply:
Thank you 🙏